### DISCUSSION TITLE
Can the log file path takes in the environment variable?

### DISCUSSION THREAD

**Discussion by LinXing1994** [bluenviron/mediamtx#2349] [Questions]
Hi,

I am setting up high availability for the application. 
In other words, the server will have few replicas with a single set of the application and config files. However, all the log files must be written to the same storage cluster but on a different path. For example, replica 1 writes to ./log/replica1/rtsp.log and replica 2 writes to ./log/replica2/rtsp.log . Hence, I need the application to read environment variables and produce log files to different paths accordingly.

And my question is: will the config file be able to do something like ./log/${enviroment_variable}/rtsp.log, so that my log files can go to different paths in my storage cluster when the server running concurrently?

**Comment by aler9** [comment]
Hello, you can set any parameter you want through environment variables as explained in the README. In this case:

```
MTX_LOGFILE=./log/replica1/rtsp.log ./mediamtx
```

[a comment by an automated account is not shown]
